app/api/dao/mentorship_relation.py

```diff
@@ -174,58 +174,58 @@ def accept_request(user_id: int, request_id: int):
             return messages.NOT_PENDING_STATE_RELATION, HTTPStatus.FORBIDDEN
 
         # verify if I'm the receiver of the request
-        if request.action_user_id == user_id:
+        if relation.action_user_id == user_id:
             return messages.CANT_ACCEPT_MENTOR_REQ_SENT_BY_USER, HTTPStatus.FORBIDDEN
 
         # verify if I'm involved in this relation
-        if not (request.mentee_id == user_id or request.mentor_id == user_id):
+        if not (relation.mentee_id == user_id or relation.mentor_id == user_id):
             return messages.CANT_ACCEPT_UNINVOLVED_MENTOR_RELATION, HTTPStatus.FORBIDDEN
 
-        my_requests = user.mentee_relations + user.mentor_relations
+        my_relations = user.mentee_relations + user.mentor_relations
 
         # verify if I'm on a current relation
-        for my_request in my_requests:
-            if my_request.state == MentorshipRelationState.ACCEPTED:
+        for my_relation in my_relations:
+            if my_relation.state == MentorshipRelationState.ACCEPTED:
                 return messages.USER_IS_INVOLVED_IN_A_MENTORSHIP_RELATION, HTTPStatus.FORBIDDEN
 
-        mentee = request.mentee
-        mentor = request.mentor
+        mentee = relation.mentee
+        mentor = relation.mentor
 
         # If I am mentor : Check if the mentee isn't in any other relation already
         if user_id == mentor.id:
-            mentee_requests = mentee.mentee_relations + mentee.mentor_relations
+            mentee_relations = mentee.mentee_relations + mentee.mentor_relations
 
-            for mentee_request in mentee_requests:
-                if mentee_request.state == MentorshipRelationState.ACCEPTED:
+            for mentee_relation in mentee_relations:
+                if mentee_relation.state == MentorshipRelationState.ACCEPTED:
                     return messages.MENTEE_ALREADY_IN_A_RELATION, HTTPStatus.BAD_REQUEST
         # If I am mentee : Check if the mentor isn't in any other relation already
         else:
-            mentor_requests = mentor.mentee_relations + mentor.mentor_relations
+            mentor_relations = mentor.mentee_relations + mentor.mentor_relations
 
-            for mentor_request in mentor_requests:
-                if mentor_request.state == MentorshipRelationState.ACCEPTED:
+            for mentor_relation in mentor_relations:
+                if mentor_relation.state == MentorshipRelationState.ACCEPTED:
                     return messages.MENTOR_ALREADY_IN_A_RELATION, HTTPStatus.BAD_REQUEST
 
         # All was checked
-        request.state = MentorshipRelationState.ACCEPTED
-        request.save_to_db()
+        relation.state = MentorshipRelationState.ACCEPTED
+        relation.save_to_db()
 
         return messages.MENTORSHIP_RELATION_WAS_ACCEPTED_SUCCESSFULLY, HTTPStatus.OK
 
     @staticmethod
     @email_verification_required
-    def reject_request(user_id: int, request_id: int):
+    def reject_request(user_id: int, relation_id: int):
```

Comment:
Same here i.e no need to do any of the changes(i.e from request_id to relation_id) in this function. Since, we still haven't accepted the relation request.